doc/help/kdb-find.md

```diff
@@ -56,7 +56,7 @@ kdb find 'fizz'
 #> user:/tests/find/tests/fizz/buzz
 #> user:/tests/find/tostfizz
 
-kdb rm -r /tests/find
+kdb rm -r user:/tests/find/
```

Comment:
Is the slash at the end important? I've never seen that.

Reply:
It shouldn't matter, I probably just forgot to remove it after trying something.